lib/src/types/navigation_initialization_params.dart

```diff
@@ -0,0 +1,24 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// Determines how application should behave when a application task is removed.
+/// {@category Navigation}
+enum TaskRemovedBehavior {
+  /// The default state, indicating that navigation guidance,
```

Comment:
Here it says that this is default, but the default is set to quiteService on interface.
I would change the default to continueService.